src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -118,9 +96,9 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
    *
    * @return ContentEntityInterface[]
    */
-  private function getUserGroups() {
-    $other_groups = Og::getEntityGroups('user', $this->getAccount()->id());
-    return $other_groups[$this->configuration['target_type']];
+  protected function getUserGroups() {
+    $other_groups = Og::getEntityGroups('user', $this->currentUser->id());
```

Comment:
@damiankloip we now have tests on 8.x-1.x that are using this, so it means we need to somehow set the `$this->currentUser`

I'm just starting to look at it, but wonder if you already had an idea how to do it the right way?